### PR TITLE
Feat/new password

### PR DESCRIPTION
**Pull request by MiguelDecode**
## Description

Create a page for a user can change the password of the account.
Add HeroLogo and some styles.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Enhancement (improvement of an existing feature)
- [x] Refactor (code change that neither fixes a bug nor adds a feature)
- [ ] Documentation (changes or additions to documentation)
- [ ] Other (please specify):

## Related Issues

https://trello.com/c/TVPydjGr

## Screenshots

image

### CODE REVIEW

File: src/components/InputPasswordForm/InputPasswordForm.tsx
Comment (MiguelDecode) [inline src/components/InputPasswordForm/InputPasswordForm.tsx:1]: I must change a bit this component to habiltate the use of ref as a prop.

File: src/pages/NewPasswordPage.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+const NewPasswordPage = () => {
+  const passwordRef = useRef<HTMLInputElement>(null);
+  const passwordCheckRef = useRef<HTMLInputElement>(null);
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const password = passwordRef.current?.value;
```
Comment (Alejandro25AR) [inline src/pages/NewPasswordPage.tsx]: The whole password validation part is handled in the PasswordInputWithRequirements component I am creating. 
imagen

File: src/components/InputPasswordForm/InputPasswordForm.tsx
Comment (Alejandro25AR) [reply src/components/InputPasswordForm/InputPasswordForm.tsx:1]: If you change the component, you will have to use it with reference everywhere, including registration, login and password recovery, in addition to all operations performed on the profile that request the password.

**Review by Alejandro25AR — COMMENTED** [review]
In a moment I will finish the PasswordInputWithRequirements component so that you can use it.

File: src/pages/NewPasswordPage.tsx (hunk 1 above)
Comment (MiguelDecode) [reply src/pages/NewPasswordPage.tsx]: That's seem more beautiful than I think. Nice work, I'm sure I can't do that.

**Comment by MiguelDecode** [comment]
I'm have a conflict with git and I try to make a rebase but I all the previous commits appear in the history and I think it's awful. Sorry for the commit history.

**Review by Alejandro25AR — APPROVED** [review]

File: src/pages/NewPasswordPage.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import { useFetch } from "../hooks/useFetch";
+import Button from "../components/Button/Button";
+import ButtonBackIcon from "../components/ButtonBackIcon/ButtonBackIcon";
+import PasswordInputWithRequirements from "../components/PasswordInputWithRequirements/PasswordInputWithRequirements";
+import HeroLogo from "../components/HeroLogo/HeroLogo";
+
+const NewPasswordPage = () => {
+  const [password, setPassword] = useState("");
+
+  const url = "http://localhost:8007/user/rpassword/reset";
```
Comment (Alejandro25AR) [inline src/pages/NewPasswordPage.tsx]: this endpoint is not yet operational

Comment (MiguelDecode) [reply src/pages/NewPasswordPage.tsx]: Yes, this endpoint not appear in the swagger i take from a trello card.

**Pull request merged**